src/metabase/models/setting.clj

```diff
@@ -526,7 +576,9 @@
          source-fns [user-local-value
                      database-local-value
                      env-var-value
-                     db-or-cache-value
+                     (if *disable-init*
+                       db-or-cache-value
+                       db-or-cache-or-init-value)
```

Comment:
this would do the same thing, but doesn't require an extra `db-or-cache-or-init-value` function
```suggestion
                     db-or-cache-value
                     (when *disable-init*
                       init!)
```

Reply:
So much nicer! Classic inverse typo - `s/when/when-not` though 😄